recipe/meta.yaml

```diff
@@ -1,40 +1,32 @@
 {% set name = "adwaita-icon-theme" %}
-{% set version = "40.1.1" %}
-{% set major_version = '.'.join(version.split('.', 1)[:1]) %}
+{% set version = "48.0" %}
+{% set major_version = version.split('.')[0] %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://download.gnome.org/sources/{{ name }}/{{ major_version }}/{{ name }}-{{ version }}.tar.xz
-  sha256: 0b6c436ed6ad9887a88ada1f72a0197b1eb73b020d8d344abab4c7fa7250f8f6
+  sha256: 847068888650d9673115be6dbf2bfdc31a46aebc528a6a9db4420e60e656b8d4
 
 build:
-  number: 1
-  # This package can be noarch on all platforms except for Windows.
-  # But, we want to keep building for linux-64, osx-64, and linux-aarch64 for
-  # now because of https://github.com/conda/conda/issues/9905 until a new noarch
-  # package would be preferred over an old arch-specific package.
-  # To get a noarch build for using on other platforms (linux-ppc64le, osx-arm64)
-  # we do an extra build on linux that makes a noarch package.
-  merge_build_host: false  # [not win]
-  # windows build assume one prefix (between e.g. python/gtk3), so use merged host/build
-  merge_build_host: true  # [win]
-  ignore_run_exports:
-    - gtk3  # [win]
-    - librsvg  # [win]
+  # gtk3 not available on windows or s390x
+  skip: true  # [win or s390x]
+  number: 0
 
 requirements:
   build:
-    # gtk3 and librsvg here for rendering svg to png, build ok because platform independent
+    # gtk needed for building icon cache, build OK because cache file is platform-independent
     - gtk3
-    - librsvg
-    - make  # [unix]
+    - meson
+    - ninja
     - python
   host:
     # explicitly empty
   run:
+    - __unix  # [unix]
+    - __win  # [win]
```

Comment:
Is it relevant only for noarch packages?

Reply:
Fixed! Thanks!